bot/kodiak/tests/fixtures.py

```diff
@@ -19,3 +19,11 @@ def create_commit(
             )
         ),
     )
+
+
+class FakeThottler:
```

Comment:
I think you can inherit from the protocol and the type checker will enforce that the shape matches

Reply:
does work :(